components/icons/close-icon.tsx

```diff
@@ -0,0 +1,13 @@
+export default function CloseIcon(): JSX.Element {
```

Comment:
Drop the "Icon" in this component name **in this file only.** When you import it in other files, you *should* add the "Icon" to the component name. But, in this file, the exported function should only be named `Close` to match the desired filename (`components/icons/close.tsx`).